hack/package_chart.sh

```
@@ -0,0 +1,56 @@
+input_dir=$1
+output_dir=$2
+tmp_dir=$3
+
+if [ "x$#" = "x4" ]
+then
+        dependency_dir=$4
+fi
+
+rm -rf "$output_dir"
+mkdir -p "$output_dir"
+cat $input_dir/versions_map | while read package version commit
```

Shellcheck (0.10.0)

[warning] version appears unused. Verify use (or export if used externally).

(SC2034)